bigchaindb/backend/mongodb/query.py

```diff
@@ -17,7 +17,10 @@
 @register_query(MongoDBConnection)
 def write_transaction(conn, signed_transaction):
     try:
-        return conn.db['backlog'].insert_one(signed_transaction)
+        return conn.run(
+            lambda db:
```

Comment:
Couldn't it be, to save a little space:

```python
return conn.run(lambda db:
    db['backlog'].insert_one(signed_transaction))
```

Another idea: Use a decorator like `@retry` and then just return the lambda directly :)